test/models/incomeSourceDetails/IncomeSourceIdSpecification.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.incomeSourceDetails
+
+import models.core.IncomeSourceIdHash
+import org.scalacheck.{Gen, Properties}
+import IncomeSourceIdHash._
+import models.core.IncomeSourceId._
+
+object IncomeSourceIdSpecification extends Properties("IncomeSourceId") {
+  import org.scalacheck.Prop.forAll
```

Comment:
Move import to top perhaps?

Reply:
not sure if this is make big difference, as its been added by IDE, move in imports just in case